src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,65 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static final String COMMA = ",";
+    public static final String DELIMETER = ",|:";
+    public static final String CUSTOMDELIMETER = "//(.)\n(.*)";
+    public static final int DEFAULT_VALUE_OF_NULL_AND_EMPTYSTRING = 0;
+    public static final Pattern pattern = Pattern.compile(CUSTOMDELIMETER);
```

Comment:
```suggestion
    private static final String DELIMITER = ",|:";
    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
    private static final int DEFAULT_VALUE_OF_NULL_AND_EMPTY_STRING = 0;
    private static final Pattern pattern = Pattern.compile(CUSTOM_DELIMITER);
```
1. 외부에 노출이 필요없는 상수인 것 같네요 :)
2. 미사용 상수는 제거해주세요 😄 
3. 오타와 띄어쓰기가 필요한 부분이 있네요